nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -98,6 +99,8 @@
 
   @Nullable protected String castToNonNullMethod;
 
+  @Nullable protected String customNullableAnnotation;
```

Comment:
How does this change relate to outputting fix information?

Reply:
This is not related to outputting fix informations. This flag is used in projects that are partially annotated with different set of custom nullable annotations. Since these annotations already exist in the source code (however not complete), they are already approved by programmers and can have significant guidance for our tool to find the solution. With this flag, we can inform NullAway of such annotations in the source code.

Reply:
Ok.  We should discuss this change separately.  Maybe put it in a separate pull request?

Reply:
Please find the link to the [PR](https://github.com/uber/NullAway/pull/522) which adds this feature. Thank you